content/public/browser/rewards_context.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CONTENT_PUBLIC_BROWSER_REWARDS_CONTEXT_H_
+#define BRAVE_CONTENT_PUBLIC_BROWSER_REWARDS_CONTEXT_H_
+
+#include <memory>
+#include <string>
+
+#include "base/callback_forward.h"
+#include "base/supports_user_data.h"
+#include "base/supports_user_data.h"
+#include "brave/content/public/browser/rewards_context.h"
+#include "chrome/browser/profiles/profile.h"
+#include "content/common/content_export.h"
+#include "content/public/browser/browser_context.h"
+
+namespace content {
+
+class RewardsDataRemover;
+class RewardsDataRemoverDelegate;
+
+// This class holds the context needed for a Rewards session.
+// It lives on the UI thread. All these methods must only be called on the UI
+// thread.
+class CONTENT_EXPORT RewardsContext : public BrowserContext {
```

Comment:
what is this class and why does it subclass BrowserContext?

Reply:
This file was removed.